### DISCUSSION TITLE
404 after working fine

### DISCUSSION THREAD

**Discussion by itzsaga** [community/community#21534] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve had my GH Pages site up for a while. I just recently realized that it’s a 404 currently. After trying everything I could think of I decided to post here. Not sure what happened to my site:</p>
<p><a href="https://github.com/itzsaga/itzsaga.github.io" rel="nofollow noopener">https://github.com/itzsaga/itzsaga.github.io</a></p>
<p>Any more obscure troubleshooting steps?</p>
<p>Thanks.</p>

**Comment by foopod** [comment]
<p>I tried to reach other site assets like images via…</p>
<p><a href="https://itzsaga.github.io/itzsaga.github.io/images/50.jpg" rel="nofollow noopener">https://itzsaga.github.io/itzsaga.github.io/images/50.jpg</a> and…</p>
<p><a href="https://itzsaga.github.io/images/50.jpg" rel="nofollow noopener">https://itzsaga.github.io/images/50.jpg</a></p>
<p>But nothing.</p>
<p>Maybe try searching your email for <em>Page build failure.</em> Any failures mean that the website won’t be published.</p>
<p>Double check the repo settings to make sure that the Github Pages source is pointing to master and right above that should be a link to the live version.</p>
<p>If that doesn’t help you can try turning Pages off and on to trigger a republish. Good Luck!</p>

**Comment by itzsaga** [comment, marked as answer]
<p>So I haven’t touched it and it works now. ¯_(ツ)_/¯</p>
<p>Thanks for taking a look.</p>
